external/pmrrr/src/plarre.c

```diff
@@ -696,7 +696,8 @@ int eigval_root_proc
 	}
       } else if (jtry == MAX_TRY_RRR-1) {
 	fprintf(stderr,"No initial representation could be found.\n");
-	exit(3);
+	return 3;
+	//exit(3);
```

Comment:
Upon further review, modulo removing the commented out line, I approve this PR, as the only callsite asserts that the return value is zero.